src/main/java/com/segment/analytics/android/integrations/firebase/FirebaseIntegration.java

```diff
@@ -122,6 +125,20 @@ public void onActivityResumed(Activity activity) {
     }
   }
 
+  @Override
+  public void onActivityStarted(Activity activity) {
+    super.onActivityStarted(activity);
+
+    this.currentActivity = activity;
+  }
+
+  @Override
+  public void onActivityStopped(Activity activity) {
+    super.onActivityStopped(activity);
+
+    this.currentActivity = null;
```

Comment:
I think `this` here will refer to onActivityStopped rather than `FirebaseIntegration`, which I think is what you actually want.

Reply:
@brennan  I added `integration.onActivityStopped(activity);` after the `integration.onActivityStarted(activity)` in the unit tests locally to inspect the values.  Looks like `this` is referring to `FirebaseIntegration`:
<img width="596" alt="Screen Shot 2020-02-20 at 2 24 16 PM" src="https://user-images.githubusercontent.com/31782219/74985703-db9b5980-53ec-11ea-94ea-61946137af0c.png">

<img width="459" alt="Screen Shot 2020-02-20 at 2 24 04 PM" src="https://user-images.githubusercontent.com/31782219/74985718-e35afe00-53ec-11ea-87d3-be71180af23a.png">